direct/src/showbase/ShowBase.py

```diff
@@ -71,6 +71,7 @@
     from direct.showbase import GarbageReport
     from direct.directutil import DeltaProfiler
     from . import OnScreenDebug
+import warnings
```

Comment:
If you indent this line by four spaces it falls under the `if __debug__:` as well :)

I don't generally suggest putting the `import warnings` under `if __debug__` but in this case it's serendipitous that there is already a block of imports under such a check.